### ISSUE TITLE
[req] Support Boolean operators in manual search filter

### ISSUE THREAD

**Issue by ilike2burnthing** [Jackett/Jackett#6791]
Not sure if this is possible, but it would be great to add support for Boolean operators when using the manual search filter (not 'Query', I am sure that would definitely be impossible, but rather the secondary 'Search' which filters the results).

It would be great to be able to filter by exact phrases, remove results containing a specific word, maybe even use wildcards (assuming that doesn't cause issues).

For example, when searching for a show called Huge, not surprisingly you get a lot of porn, and filtering by '2010' just gives you a lot of 2010 porn instead. Being able to add -xxx or ~huge?2010 to the search filter would definitely be helpful.

Thanks in advance!

**Comment by RoloSoze** [comment]
@garfield69 I'm assuming #8048 closes this as well? Or does it only partially address it?

**Comment by garfield69** [comment]
8048 addresses this issue as far as I am concerned.

**Comment by ilike2burnthing** [comment]
@RoloSoze see - https://github.com/Jackett/Jackett/pull/8048#issuecomment-609449578

Doesn't seem possible at the moment DataTables doesn't play nice with `-` or `+`, so it would throw a fit at wildcards 😋 nah, just wouldn't work

**Comment by RoloSoze** [comment]
> 
> 
> 8048 addresses this issue as far as I am concerned.

@garfield69 Great. As a side note, I just took a quick look at all open issues and mentioned you wherever I thought an open issue might've been addressed already. However, you know Jackett best, and I'm sure that if you get the chance to take a quick scan yourself, you'll be able to instantly close a lot of older issues that have been addressed, cannot be addressed, or where there was no response from the OP. Fewer valid open issues is probably best.
